shared/security/src/main/java/io/pravega/shared/security/auth/PermissionsHelper.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.shared.security.auth;
+
+import io.pravega.auth.AuthHandler;
+import lombok.NonNull;
+
+/**
+ * Helper methods for permissions.
+ */
+public class PermissionsHelper {
+
+    /**
+     * Translates the specified {@code accessOperation} to an {@link AuthHandler.Permissions} object.
+     *
+     * @param accessOperation accessOperation to translate
+     * @return a {@link AuthHandler.Permissions} object that represents the specified {@code accessOperation}
+     */
+    public static AuthHandler.Permissions toAuthHandlerPermissions(AccessOperation accessOperation) {
+        if (accessOperation.equals(AccessOperation.READ)) {
+            return AuthHandler.Permissions.READ;
+        } else if (accessOperation.equals(AccessOperation.WRITE) ||
+                accessOperation.equals(AccessOperation.READ_UPDATE)) {
+            return AuthHandler.Permissions.READ_UPDATE;
+        } else {
+            throw new IllegalArgumentException("Cannot translate");
```

Comment:
Include the argument too. IT will help with debugging.

Reply:
Yeah, that was a miss. Fixed.